x/evidence/client/cli/query.go

```diff
@@ -0,0 +1,115 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/evidence/internal/types"
+)
+
+const (
+	flagPage  = "page"
```

Comment:
nit, we should move these to `clint/flags/flags.go` as almost all modules use them